azure/services/agentpools/spec.go

```diff
@@ -31,6 +31,9 @@ import (
 
 // AgentPoolSpec contains agent pool specification details.
 type AgentPoolSpec struct {
+	// AdditionalTags is an optional set of tags to add to Azure resources managed by the Azure provider, in addition to the ones added by default.
+	AdditionalTags infrav1.Tags
```

Comment:
I changed this from my previous version using the `type` defined in `infrav1.Tags`.
Is this the correct approach ?